pkg/cli/cmds/root.go

```diff
@@ -25,6 +25,18 @@ var (
 			EnvVars:     []string{"RKE2_REPO"},
 			Destination: &config.Repo,
 		},
+		&cli.StringFlag{
```

Comment:
Are these dependent on one another? Should we have a check to see if both are present and not just one?

Reply:
well it depends, so cloud config doesnt work without cloud provider name, but the opposite is not true, so you can pass only cloud provider name  for some providers without the need to set up the cloud config file

Reply:
I will add a check if the config is supported without the name